src/storage/store/account/onchain_event_store.rs

```diff
@@ -425,6 +425,34 @@ impl OnchainEventStore {
         Ok(onchain_events)
     }
 
+    pub fn get_all_onchain_events(
```

Comment:
Let's make fid optional on the existing `get_onchain_events` function.

Reply:
Ok done-zo 😁